genie-web/src/main/java/com/netflix/genie/web/services/JobMetricsService.java

```diff
@@ -37,4 +37,13 @@
      * @return The total memory used by jobs in megabytes
      */
     int getUsedMemory();
+
+    /**
+     * Get the number of agents currently connected to this server.
+     *
+     * @return The number of agents connected to the server
+     */
+    default long getNumActiveAgents() {
+        return 0L;
```

Comment:
Why the default value for this one?

Reply:
Just to make it easier if something didn't want to implement all of them. It was a remnant of when I had it in the `JobMetricsService` and didn't want to implement it in 2 or 3 places. Doesn't really harm anything.